setup.py

```diff
@@ -0,0 +1,4 @@
+from setuptools import setup
+
+if __name__ == '__main__':
+    setup()
```

Comment:
Complete?

Reply:
Yup it will loke automatically in the setup.cfg file